### PR TITLE
OBSDOCS-1284: Logging 5.9.6 Release Notes

### PR DESCRIPTION
**Pull request by shreyasiddhartha**
Change type: Doc update; Logging Z-Stream Release Notes - 5.9.6
Doc JIRA: https://issues.redhat.com/browse/OBSDOCS-1284

Fix Version: 4.13, 4.14, 4.15, 4.16

Doc Preview: https://81473--ocpdocs-pr.netlify.app/openshift-dedicated/latest/observability/logging/logging_release_notes/logging-5-9-release-notes.html#cluster-logging-release-notes-5-9-6_logging-5-9-release-notes

SME Review: @periklis 
QE Review: @kabirbhartiRH 
Peer Review: @ShaunaDiaz

### CODE REVIEW

**Review by periklis — COMMENTED** [review]
LGTM

File: modules/logging-release-notes-5-9-6.adoc [hunk 1]
```diff
@@ -0,0 +1,31 @@
+// module included in logging-5-9-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="cluster-logging-release-notes-5-9-6_{context}"]
+= Logging 5.9.6
+This release includes a link:https://access.redhat.com/errata/RHBA-2024:6095[OpenShift Logging Bug Fix Release 5.9.6].
+
+[id="openshift-logging-5-9-6-bug-fixes_{context}"]
+== Bug fixes
+
+* Before this update, the collector deployment ignored secret changes, causing receivers to reject logs. With this update, the system rolls out a new pod when there is a change in the secret value, ensuring the collector reloads the updated secrets. (link:https://issues.redhat.com/browse/LOG-5525[LOG-5525])
```
Comment (ShaunaDiaz) [inline modules/logging-release-notes-5-9-6.adoc:10]:
```suggestion
* Before this update, the collector deployment ignored secret changes, causing receivers to reject logs. With this update, the system rolls out a new pod when there is a change in the secret value, ensuring that the collector reloads the updated secrets. (link:https://issues.redhat.com/browse/LOG-5525[LOG-5525])
```

File: modules/logging-release-notes-5-9-6.adoc [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+* Before this update, the drop filter could not handle non-string values (e.g., `.responseStatus.code: 403`). With this update, the drop filter now works properly with these values. (link:https://issues.redhat.com/browse/LOG-5815[LOG-5815])
+
+* Before this update, the collector used the default settings to collect audit logs, without handling back the load from output receivers. With this update, the process for collecting audit logs has been improved to better manage file handles and log reading efficiency. (link:https://issues.redhat.com/browse/LOG-5866[LOG-5866])
+
+* Before this update, the `must-gather` tool failed on clusters with non-AMD64 architectures like Azure Resource Manager (ARM) or PowerPC. With this update, the tool now detects the cluster's architecture at runtime and uses architecture-independent paths and dependencies, allowing it to run smoothly on platforms like ARM and PowerPC. (link:https://issues.redhat.com/browse/LOG-5997[LOG-5997])
```
Comment (ShaunaDiaz) [inline modules/logging-release-notes-5-9-6.adoc:18]:
```suggestion
* Before this update, the `must-gather` tool failed on clusters with non-AMD64 architectures such as Azure Resource Manager (ARM) or PowerPC. With this update, the tool now detects the cluster architecture at runtime and uses architecture-independent paths and dependencies. The detection allows `must-gather` to run smoothly on platforms like ARM and PowerPC. (link:https://issues.redhat.com/browse/LOG-5997[LOG-5997])
```

File: modules/logging-release-notes-5-9-6.adoc [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+* Before this update, the log level was set using a mix of structured and unstructured keywords that was unclear. With this update, the log level follows a clear, documented order, starting with structured keywords and then moving to unstructured ones, if needed. (link:https://issues.redhat.com/browse/LOG-6016[LOG-6016])
```
Comment (ShaunaDiaz) [inline modules/logging-release-notes-5-9-6.adoc:20]:
```suggestion
* Before this update, the log level was set using a mix of structured and unstructured keywords that were unclear. With this update, the log level follows a clear, documented order, starting with structured keywords. (link:https://issues.redhat.com/browse/LOG-6016[LOG-6016])
```
Fairly obvious that unstructured would come next. Are there any other type of keywords? If so, just saying where it starts is a bit more evergreen.

Comment (shreyasiddhartha) [reply modules/logging-release-notes-5-9-6.adoc:20]: I agree, but I couldn't find any other keyword related to it. I'm using your suggestion.

File: modules/logging-release-notes-5-9-6.adoc [hunk 4]
```diff
@@ -0,0 +1,31 @@
+// module included in logging-5-9-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="cluster-logging-release-notes-5-9-6_{context}"]
+= Logging 5.9.6
+This release includes a link:https://access.redhat.com/errata/RHBA-2024:6095[OpenShift Logging Bug Fix Release 5.9.6].
```
[the remark this answers is not in the document]
Comment (shreyasiddhartha) [reply modules/logging-release-notes-5-9-6.adoc:5]: I am removing the article, as I used to do before, if that's okay.

**Comment by ShaunaDiaz** [comment]
@shreyasiddhartha You can prepare a manual cherry picks that resolves the conflicts and we can merge it up!

**Comment by shreyasiddhartha** [comment]
> @shreyasiddhartha You can prepare a manual cherry picks that resolves the conflicts and we can merge it up!

Hi @ShaunaDiaz! 5.9.6 shall not be merged to 4.12. So, no manual CPs needed. :)

**Pull request merged**